build-tools/libzip/libzip.mdproj

```diff
@@ -13,9 +13,8 @@
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
     <OutputPath>..\..\bin\$(Configuration)</OutputPath>
   </PropertyGroup>
-  <Import Project="$(MSBuildBinPath)\Microsoft.CSharp.targets" />

```

Comment:
I believe we need to import *something*, as `libzip.targets` uses `$(IntermediateOutputPath)`/etc. which won't be defined unless `Microsoft.*.targets` is imported.

Perhaps import `Microsoft.Common.targets`? That's what `src/monodroid/monodroid.mdproj` imports, among others.